src/System.Net.Http/src/System/Net/Http/SocketsHttpHandler/Http2Connection.cs

```diff
@@ -701,6 +701,19 @@ private void WriteHeader(string name, string value)
             _headerBuffer.Commit(bytesWritten);
         }
 
+        private void WriteHeader(HeaderDescriptor header, string[] value, string separator)
```

Comment:
For consistency with the above routine, seems like this should take `string name` instead of `HeaderDescriptor header`.